.env.example

```
@@ -227,6 +227,8 @@ TINYAUTH_LDAP_GROUPCACHETTL=900
 
 # Enable the OAuth bridge, uses a new way to format OAuth user information.
 TINYAUTH_EXPERIMENTAL_OAUTHBRIDGEENABLED=false
+# Disable the fallback to forward_auth modules when auth_request or ext_authz fail.
+TINYAUTH_EXPERIMENTAL_DISABLEAUTHMODULEFALLBACK=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TINYAUTH_EXPERIMENTAL_DISABLEAUTHMODULEFALLBACK key should go before the TINYAUTH_EXPERIMENTAL_OAUTHBRIDGEENABLED key

(UnorderedKey)